src/components/Header.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
 import logo from '../images/Logo.svg';
+import menu from '../images/align-justify-solid.svg';
+import times from '../images/times-solid.svg';
 import Button from './Button';
+import Container from './Container';
 
 const Wrapper = styled.header`
   padding: 25px 0;
+
+  @media screen and (max-width: 800px) {
+    position: relative;
+    border-bottom: 1px solid #bcd0e5;
+  }
```

Comment:
Please convert to mobile-first style (you should mostly use `min-width` instead of `max-width`):

```css
  position: relative;
  padding: 25px 0;
  border-bottom: 1px solid #bcd0e5;
  
  @media screen and (min-width: 800px) {
    border-bottom: 0;
  }
```

Reply:
Done